includes/store-minidumps-as-attachments-configuration.mdx

```diff
@@ -0,0 +1,15 @@
+### Enabling Minidump Storage
+
+You can enable _Store Minidumps As Attachments_ in your organization or project settings under **Security & Privacy**. By default, this setting is disabled. Determine the maximum number of crash reports that will be stored per issue; disabled, unlimited, or maximum per issue:
+
+<Include name="common-imgs/store-minidumps-as-attachments" />
+
+If you set a limit per issue, as in the example above, a limit of 5, Sentry will store the first 5 attachments associated with this issue, but drop any that follow. To make room for additional attachments, delete them. Sentry will then accept attachments until the limit is reached again.
+
+### Access to Attachments
+
+To limit access to attachments, navigate to your organization's **General Settings**, then select the _Attachments Access_ dropdown to set appropriate access — any member of your organization, the organization billing owner, member, admin, manager, or owner.
+
+<Include name="common-imgs/attachments-access" />
+
+By default, access is granted to all members when storage is enabled. If a member does not have access to the project, the ability to download an attachment is not available; the button will be greyed out in Sentry. The member may only view that an attachment is stored.
```

Comment:
The access to Attachments is not specific to Minidumps. Can we keep that separate?